DEVELOPMENT.md

```diff
@@ -16,40 +16,81 @@
 
 ## Running Locally
 
-Step 1
-```bash
-# Clone the repo
-git clone https://github.com/tektoncd/website && cd website
-```
+You can install the dependencies locally on your computer or use [docker
+compose][docker-compose] to run the included development image.
 
-Step 2
-```bash
-# Install node modules
-npm install
-```
+[docker-compose]: https://docs.docker.com/compose/install/
 
-Step 3
-```bash
-Install the sync script (https://github.com/tektoncd/website/blob/master/sync/README.md)
-python3 -m venv .venv
-source .venv/bin/activate    
-pip3 install -r requirements.txt
+### Running in a Docker container
 
-```
-Step 4
-```bash
-# Run the sync.
-# This clones docs repositories to local cache and builds the
-# documentation content for the website
-./sync/sync.py
-```
+1. Build the Docker image
+
+   ```bash
+   docker-compose build
+
+   ```
+
+1. Run the built image
+
+   ```bash
+   docker-compose up
+   ```
+
+1. Verify that the website is working
+
+   Open your web browser and type `http://localhost:8888` in the navigation bar.
+   This opens a local instance of the website, you can now make changes in the
+   documentation and those changes will immediately show up in the browser after
+   you save.
+
+To remove the produced images run:
 
-Step 5
 ```bash
-# Build and serve the website locally
-netlify dev
+docker-compose rm
 ```
 
+### Running Natively
+
+1. Clone the repository
+
+   ```bash
+   git clone https://github.com/tektoncd/website && cd website
+   ```
+
+1. Install the required node modules
+
+   ```bash
+   npm install
+   ```
+
+1. Install the dependencies for the [sync script](https://github.com/tektoncd/website/blob/master/sync/README.md)
+
+   ```bash
+   python3 -m venv .venv
+   source .venv/bin/activate    
+   pip3 install -r requirements.txt
+
+   ```
+
+1. Run the sync script
+
+   ```bash
+   ./sync/sync.py
+   ```
+
+1. Serve the website locally
+
+   ```bash
+   netlify dev
+   ```
+
+1. Verify that the website is working
+
+   Open your web browser and type `http://localhost:8888` in the navigation bar.
+   This opens a local instance of the website, you can now make changes in the
+   documentation and those changes will immediately show up in the browser after
+   you save.
+
```

Comment:
totally some scope creep but if we could just get `netlify dev` to work from the docker container as well we could remove this whole section.

on the other hand, running the hugo server is probably good enough, and maybe with the new docker based instructions we could just have a reference to `netlify dev` as another option and remove the section anyway?

Reply:
I can update the Dockerfile in a subsequent PR to make this work.